src/LRS.js

```diff
@@ -842,6 +842,7 @@ TinCan client library
             @param {Function} [cfg.callback] Callback to execute on completion
                 @param {Object|Null} cfg.callback.error
                 @param {TinCan.State|null} cfg.callback.result null if state is 404
+            @param {Object} [cfg.optHeaders] Optional object containing additional headers to add to request
```

Comment:
I don't think we need to include 'opt' in the name, I think it is fine to just stick with 'headers'. The `[]` in the API doc already indicates that it is optional so no need to have it in the name or even the document string.